lib/cli.js

```diff
@@ -46,6 +46,13 @@ function processArgs (argv, options, fs, path) {
     options.failOnEmptyTestSuite = options.failOnEmptyTestSuite === 'true'
   }
 
+  if (helper.isString(options.failOnFailingTestSuite)) {
+    options.failOnFailingTestSuite = options.failOnFailingTestSuite === 'true'
+  } else if (typeof options.failOnFailingTestSuite === 'undefined') {
+    // Default value to true, to avoid chanching application default behaviour
```

Comment:
what is chanching?

Reply:
I want to keep karma exit with failure unless the new argument is set, as it did before. In new version of the patch I updated the logic using the flag instead, so no default value is needed in `cli.js` to achieve this